src/microcontroller/__init__.py

```diff
@@ -74,7 +74,9 @@ def __repr__(self):
     from adafruit_blinka.microcontroller.nxp_lpc4330 import *
 elif chip_id == ap_chip.MIPS24KC:
     from adafruit_blinka.microcontroller.atheros.ar9331.pin import *
-elif chip_id == ap_chip.MIPS24KEC:
-    from adafruit_blinka.microcontroller.mips24kec.pin import *
```

Comment:
Please add the MIPS24KEC code section back in. Otherwise it will break Onion Omega 2 support.

Reply:
I did not remove any code, at least not on purpose, but I know what happened. That code was (is) right above where I added the ryzen code. I cloned off the MIPS24KEC code to add the ryzen stuff and forgot to put the original code back. All restored!

Reply:
Thanks!